src/doc/DocAnnotator.js

```diff
@@ -594,12 +597,7 @@ class DocAnnotator extends Annotator {
             this.modeControllers[TYPES.highlight_comment].applyActionToThreads((thread) => thread.reset(), page);
         }
 
-        let mouseEvent = event;
-        // $FlowFixMe
-        if (this.hasTouch && event.targetTouches) {
-            mouseEvent = event.targetTouches[0];
-        }
-        this.lastHighlightEvent = mouseEvent;
+        this.lastHighlightEvent = this.hasTouch ? get(event, 'targetTouches[0]', event) : event;
```

Comment:
If there's no touch then target touches won't exist. I think you could just do:

this.lastHighlightEvent = get(event, 'targetTouches[0], event);